components/Dashboard/TransactionTable.vue

```diff
@@ -64,19 +271,74 @@ export default {
     await axios
       .get("https://pwcdemo-1c4d3.firebaseio.com/users.json")
       .then(response => {
-        (this.loading = false), (this.facts = Object.values(response.data));
+        const postArray = [];
+        for (const key in response.data) {
+          postArray.push({ ...response.data[key], id: key });
+        }
+        (this.loading = false), (this.facts = postArray);
+        console.log(this.facts);
+
+        // console.log(this.facts[0]);
       });
   },
   methods: {
+    fetchDataFromTable(index, factID) {
+      this.sizeFormT = this.facts[index];
+      //    this.sizeFormT = this.sizeForm;
+      this.jsonID = this.facts[index].id;
+    },
+    UpdateForm() {
+      this.$refs.sizeFormT.validate(valid => {
+        if (valid) {
+          axios
+            .put(
+              "https://pwcdemo-1c4d3.firebaseio.com/users/" +
```

Comment:
You can use template literals here. See https://developers.google.com/web/updates/2015/01/ES6-Template-Strings

Reply:
+1

Reply:
I used es6 template this problem.